v1.0-known-limitations.md

```diff
@@ -0,0 +1,68 @@
+~~~
+
+## Repeated commands in the SQL shell history
+
+In the [built-in SQL shell](use-the-built-in-sql-client.html), each previously executed command is stored in the SQL shell history. In some cases, commands are unexpectedly duplicated in the history.
+
+## Using `\|` to perform a large input in the SQL shell
+
+In the [built-in SQL shell](use-the-built-in-sql-client.html), using the [`\|`](use-the-built-in-sql-client.html#sql-shell-commands) operator to perform a large number of inputs from a file can cause the server to close the connection. This is because `\|` sends the entire file as a single query to the server, which can exceed the upper bound on the size of a packet the server can accept from any client (16MB).
+
+As a workaround, [execute the file from the command line](use-the-built-in-sql-client.html#execute-sql-statements-from-a-file) with `cat data.sql | cockroach sql` instead of from within the interactive shell.
+
+## New values generated by `DEFAULT` expressions during `ALTER TABLE ADD COLUMN`
+
+When executing an [`ALTER TABLE ADD COLUMN`](add-column.html) statement with a [`DEFAULT`](default-value.html) expression, new values generated:
+
+- use the default [search path](sql-name-resolution.html#search-path) regardless of the search path configured in the current session via `SET SEARCH_PATH`.
+- use the UTC time zone regardless of the time zone configured in the current session via [`SET TIME ZONE`](set-time-zone.html).
+- have no default database regardless of the default database configured in the current session via [`SET DATABASE`](set-database.html).
+- use the transaction timestamp for the `statement_timestamp()` function regardless of the time at which the `ALTER` statement was issued.
+
+## Load-based lease rebalancing in uneven latency deployments
+
+When nodes are started with the [`--locality`](start-a-node.html) flag, CockroachDB attempts to place the replica lease holder (the replica that client requests are forwarded to) on the node that is closest to the source of the request. This means that as client requests move geographically, so does the replica lease holder.
+
+However, you might see increased latency caused by a continuously high rate of lease transfers between datacenters in the following case:
+
+- Your cluster is running in datacenters that are very different distances away from each other.
+- Each node was started with a single tier of `--locality`, e.g., `--locality=datacenter=a`.
+- Most client requests get sent to a single datacenter because that's where all your application traffic is.
+
+To detect that this is happening, open the [Admin UI](explore-the-admin-ui.html), select the **Queues** dashboard, hover over the **Replication Queue** graph, and check the **Leases Transferred / second** data point. If that data point is larger than 0, you should consider stopping and restarting each node with additional tiers of locality to improve request latency.
```

Comment:
It's not necessarily bad if the data point is larger than 0 once in a while. The issue is if it's continuously/consistently greater than 0, indicating that leases are constantly being transferred. The occasional transfer is fine.